articles/active-directory/develop/msal-migration.md

```diff
@@ -29,6 +29,15 @@ If any of your applications use the Azure Active Directory Authentication Librar
 
 ## Why switch to MSAL?
 
+To understand 'Why MSAL?', it's important to first understand the differences between Microsoft identity platform (v2.0) and Azure Active Directory (v1.0) endpoints. The v1.0 endpoint is used by ADAL while the v2.0 endpoint is used by MSAL. If you have developed apps against the v1 endpoint in the past, you would probably be using ADAL (Azure AD authentication Library). Since the v2 endpoint has changed significantly enough, the new library, Microsoft Authentication Library (MSAL), was built for the new endpoint entirely. 
```

Comment:
```suggestion
To understand 'Why MSAL?', it's important to first understand the differences between Microsoft identity platform (v2.0) and Azure Active Directory (v1.0) endpoints. The v1.0 endpoint is used by Azure AD Authentication Library (ADAL) while the v2.0 endpoint is used by Microsoft Authentication Library (MSAL). If you've developed apps against the v1.0 endpoint in the past, you're likely using ADAL. Since the v2.0 endpoint has changed significantly enough, the new library (MSAL) was built for the new endpoint entirely. 
```